src/Gating.Example/Program.cs

```diff
@@ -128,6 +128,12 @@ internal class SampleGatedRequest : IGatedRequest
 		};
 
 
+		public HashSet<string> CloudContexts => new HashSet<string>
```

Comment:
>public [](start = 2, length = 6)

As a next PR, can you please update the documentation to reflect this change? Feel free to add an issue to track that.